src/app/modules/issue-vc/new-issue-vc/new-issue-vc.component.html

```diff
@@ -0,0 +1,62 @@
+<h4 mat-dialog-title class="mb-0 d-flex justify-content-between align-items-center">
+  Create VC
+  <button mat-icon-button mat-dialog-close>
+    <mat-icon>close</mat-icon>
+  </button>
+</h4>
+
+<div mat-dialog-content>
+  <div class="mb-2 mt-4 mx-4 mx-lg-4">
+    <form role="form" [formGroup]="form">
+      <div class="row">
+        <div class="col-lg-12">
+          <mat-label class="ml-3 pl-1">Type</mat-label>
+          <mat-form-field appearance="outline"
+                          class="mr-0 mr-lg-0 mb-2 mb-lg-0">
+            <mat-select formControlName="type" placeholder="Select Type" data-qa-id="select-type">
+              <mat-option [value]="'manufacturer'" data-qa-id="manufacturer">Manufacturer</mat-option>
+              <mat-option [value]="'owner'" data-qa-id="owner">Owner</mat-option>
+              <mat-option [value]="'installer'" data-qa-id="installer">Installer</mat-option>
+              <mat-option [value]="'verifier'" data-qa-id="verifier">Verifier</mat-option>
```

Comment:
Instead of hardcoding these values, could you obtain them dynamically using the method in this PR? https://github.com/energywebfoundation/iam-client-lib/pull/315